script/1_0_2current_en_script/Arcueid/Day 6/06_00_ARC06_1B.txt

```diff
@@ -1,40 +1,40 @@
 <Page1>
 Hisui is waiting for me in the hallway.
-My left leg is hurt so badly that I want to scream after every step.
-Although my entire body feels heavy, it's only a slight muscular pain. It's not as painful as it seems.
+My left leg is hurt so badly that every step I take makes me want to scream.
+Although my entire body feels heavy, it's only a slight muscle ache. It's not actually all that painful.
 <Page2>
-I confirmed with my hands that the cuts I received from Vlov's ice fragments on areas such as my cheeks and shoulders had completely recovered.
-The unnatural lack of even a scar makes it clear that it was the work of Arcueid. She did say that she would treat me after all, that woman.
+I felt around with my hands to confirm, but it seems that the cuts I received on my cheeks and shoulders from Vlov's ice shards have healed completely.//手で確認したが、ヴローヴの氷の破片で受けた頬や肩の切り傷は完治していた。
+The unnatural lack of any scars makes it clear that this is the result of Arcueid's handiwork. She did say that she would treat me after all, that woman.//傷跡さえない不自然さから、アルクェイドの仕業なのは明白だ。手当てをする、とか言ってたし、あいつ。
```

Comment:
> clear that this is Arcueid's handiwork.



this is a little bit J-E imo. Either naturalize with a soft swear word or drop imo
>, that woman.